app/pages/PageWrapper.spec.js

```diff
@@ -19,9 +18,9 @@ describe('pages/PageWrapper', () => {
     );
   }
 
-  test('renders DocumentTitle component', () => {
-    const title = getWrapper().find(DocumentTitle);
-    expect(title.prop('title')).toBe('Test title - Varaamo');
+  test('renders Helmet title', () => {
```

Comment:
as Helmet is not easily render a component which have prop, have to use `mount` instead of `shallow` rendering.

The code below did the same thing needed to test